v2/pkg/protocols/ssl/ssl.go

```diff
@@ -206,6 +206,11 @@ func (request *Request) ExecuteWithResults(input *contextargs.Context, dynamicVa
 
 	data := make(map[string]interface{})
 
+	payloadFromOption := generators.BuildPayloadFromOptions(request.options.Options)
```

Comment:
They should be added [here](https://github.com/projectdiscovery/nuclei/blob/63150a8ee42327f685025c4f819d11e6d956a784/v2/pkg/protocols/ssl/ssl.go#L148) to be usable before the request is sent out. Later on, they will be automatically copied in the response map [here](https://github.com/projectdiscovery/nuclei/blob/63150a8ee42327f685025c4f819d11e6d956a784/v2/pkg/protocols/ssl/ssl.go#L221)